Sources/LogBuilder.swift

```diff
@@ -9,23 +9,24 @@
 /**
  The class for building a log string.
  */
-final class LogBuilder {
+class LogBuilder {
     
     // MARK: - Properties
     
-    private static let dateFormatter: DateFormatter = {
-        var formatter = DateFormatter()
-        formatter.dateStyle = .medium
-        formatter.timeStyle = .medium
-        formatter.locale = Locale(identifier: "ja_JP")
-        
-        return formatter
-    }()
+    private let dateFormatter: DateFormatter
     
     // MARK: - Lifecycle
     
-    private init() {
-        // Singleton
+    // TODO: Remove a default argument
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Scior/Linna/blob/1e5a9e880d55a2046af30e50ad421b50fd3a5651/Sources/LogBuilder.swift#L20"></span>TODOs should be resolved (Remove a default argument).</td>
    </tr>
  </tbody>
</table>